### PR TITLE
fix: swatch introspection on Style control

### PR DESCRIPTION
**Pull request by fikrikarim**

### CODE REVIEW

File: packages/runtime/src/controls/style.ts [hunk 1]
```diff
@@ -219,6 +219,19 @@ export class StyleControl extends PropController<StyleControlMessage> {
...
 
+export function getStyleSwatchIds(data: StyleControlData): string[] {
+  const swatchIds: string[] = []
+
+  data?.border?.forEach(({ value }) => {
+    if (value.borderTop?.color?.swatchId) swatchIds.push(value.borderTop.color.swatchId)
+    if (value.borderRight?.color?.swatchId) swatchIds.push(value.borderRight.color.swatchId)
+    if (value.borderBottom?.color?.swatchId) swatchIds.push(value.borderBottom.color.swatchId)
+    if (value.borderLeft?.color?.swatchId) swatchIds.push(value.borderLeft.color.swatchId)
+  })
```
Comment (migueloller) [inline packages/runtime/src/controls/style.ts:230]: Do we have a `getSwatchIds` function for border data that we can reuse here?

**Review by migueloller — DISMISSED** [review]
LGTM 👍🏻

File: packages/runtime/src/controls/style.ts (hunk 1 above)
Comment (fikrikarim) [reply packages/runtime/src/controls/style.ts]: Good idea. Updated.

**Review by arvinpoddar — APPROVED** [review]
Looking good

**Pull request merged**